maintainers/working_groups/learning_journey/documentation-structure.md

```diff
@@ -0,0 +1,24 @@
+- Tutorials
+    - Nix
+    - NixOS
+- How-To
+    - Language guides
+    - Workflows
+        - Nix REPL
+        - IDE integration
+        - CI
+        - Operations
+        - Containers
+        - Shells
+    - Package management
+    - Contributing
+- Explanation
+    - Nix store
+    - Cross-compilation
+    - Packaging conventions
+- Reference
+    - Nix language reference
+    - Commands reference
+    - Derivations
+    - Nix built-ins
+    - Nixpkgs lib
```

Comment:
```suggestion
    - Nixpkgs lib
    - Necessary unix concepts and familiarity before starting with nix
```

Feel free to rephrase as you see fit, I hope and think that it's clear what is meant.

Reply:
Also covering that is clearly out of scope, but mentioning it in the right places as an "external dependency" is definitely worthwhile.

Reply:
Yes, for me this goes in the same direction as what @fricklerhandwerk mentions here below about the reference manuals. 

> These should just be links to the reference manuals. They are self-contained and should be under curation of the code maintainers. That's how we do it right now and there is little reason (or possibility or capacity, in fact) to change that.

It's definitely out of scope for us to write this. 
What may be in scope for us is to link to resources that we think do a good job curating and delivering this information.  (in a similar manner and for similar reasons that we link to the Nix reference manuals)